src/plone/restapi/tests/test_documentation.py

```diff
@@ -608,3 +608,23 @@ def test_documentation_navigation(self):
         response = self.api_session.get(
             '{}/@components/navigation'.format(self.document.absolute_url()))
         save_request_and_response_for_docs('navigation', response)
+
+    def test_documentation_principals(self):
```

Comment:
@tisto I started to document this as well, but documentation wise, I think it will be pretty much redundant. So for now I will leave it as it is.